assets/out

```diff
@@ -229,6 +229,27 @@ echo "$params" | jq -c '.commands[]' | while read -r options; do
     printf '\e[92m[INFO]\e[0m Executing \e[33m%s\e[0m: %s\n' "$command" "$app_name"
     cf_target "$org" "$space"
     cf_bind_service "$app_name" "$service_instance" "$configuration"
+  elif [ "create-service-broker" = "$command" ]; then
+    broker_name=$(echo $options | jq -r '.broker_name //empty')
+    username=$(echo $options | jq -r '.username //empty')
+    password=$(echo $options | jq -r '.password //empty')
+    url=$(echo $options | jq -r '.url //empty')
+    is_space_scoped=$(echo $options | jq -r '.space_scoped //"false"')
+    printf '\e[92m[INFO]\e[0m Executing \e[33m%s\e[0m: %s %s %s %s %s\n' "$command" "$broker_name" "$username" "$password" "$url"
```

Comment:
To be consistent with other log messages, this should only output the command and the broker name (and definitely not the password)

Reply:
Bah, i'll go ahead and change this :-)